api/datadoghq/v1alpha2/datadogpodautoscaler_types.go

```diff
@@ -0,0 +1,541 @@
+	Replicas int32 `json:"desiredReplicas"`
+}
+
+// DatadogPodAutoscalerHorizontalAction represents an horizontal action done by the controller
+type DatadogPodAutoscalerHorizontalAction struct {
+	// Time is the timestamp of the action
+	Time metav1.Time `json:"time"`
+
+	// FromReplicas is the number of replicas before the action
+	FromReplicas int32 `json:"replicas"`
+
+	// ToReplicas is the effective number of replicas after the action
+	ToReplicas int32 `json:"toReplicas"`
+
+	// RecommendedReplicas is the original number of replicas recommended by Datadog
+	RecommendedReplicas *int32 `json:"recommendedReplicas,omitempty"`
+
+	// LimitedReason is the reason why the action was limited (ToReplicas != RecommendedReplicas)
+	LimitedReason *string `json:"limitedReason,omitempty"`
+}
+
+// DatadogPodAutoscalerVerticalStatus defines the status of the vertical scaling
+type DatadogPodAutoscalerVerticalStatus struct {
+	// Target is the current target of the vertical scaling
+	Target *DatadogPodAutoscalerVerticalTargetStatus `json:"target,omitempty"`
+
+	// LastAction is the last successful action done by the controller
+	LastAction *DatadogPodAutoscalerVerticalAction `json:"lastAction,omitempty"`
+}
+
+// DatadogPodAutoscalerVerticalTargetStatus defines the current target of the vertical scaling
+type DatadogPodAutoscalerVerticalTargetStatus struct {
+	// Source is the source of the value used to scale the target resource
+	Source DatadogPodAutoscalerValueSource `json:"source"`
+
+	// GeneratedAt is the timestamp at which the recommendation was generated
+	GeneratedAt metav1.Time `json:"generatedAt,omitempty"`
+
+	// Version is the current version of the received recommendation
+	Version string `json:"version"`
+
+	// Scaled is the current number of PODs having desired resources
+	Scaled *int32 `json:"scaled,omitempty"`
+
+	// DesiredResources is the desired resources for containers
+	DesiredResources []DatadogPodAutoscalerContainerResources `json:"desiredResources"`
+
+	// PODCPURequest is the sum of CPU requests for all containers (used for display)
+	PODCPURequest resource.Quantity `json:"podCPURequest"`
+
+	// PODMemoryRequest is the sum of memory requests for all containers (used for display)
+	PODMemoryRequest resource.Quantity `json:"podMemoryRequest"`
+}
+
+// DatadogPodAutoscalerVerticalActionType represents the type of action done by the controller
+type DatadogPodAutoscalerVerticalActionType string
+
+const (
+	// DatadogPodAutoscalerRolloutTriggeredVerticalActionType is the action when the controller triggers a rollout of the targetRef
+	DatadogPodAutoscalerRolloutTriggeredVerticalActionType DatadogPodAutoscalerVerticalActionType = "RolloutTriggered"
+)
+
+// DatadogPodAutoscalerVerticalAction represents a vertical action done by the controller
+type DatadogPodAutoscalerVerticalAction struct {
+	// Time is the timestamp of the action
+	Time metav1.Time `json:"time"`
+
+	// Version is the recommendation version used for the action
```

Comment:
```suggestion
	// Version is the recommended version used for the action
```

Reply:
The current version is correct - it's version numer of the recommendation. I reworded to clarify